airbyte-integrations/connectors/source-freshcaller/poetry.lock

```diff
@@ -0,0 +1,1309 @@
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+
+[[package]]
+name = "airbyte-cdk"
+version = "1.8.0"
```

Comment:
I think I know what's going on.

When we run acceptance tests and see that two new incremental tests are broken, it's very likely because we inadvertedly upgraded the CDK version. 

When you switch to Poetry, it does `poetry lock` and locks to the _current_ version that works with whatever pin you have. Here, you have `1.8.0`. When we run tests and compare to the previous version of the connector, we compare to whatever image we've built long time ago.

I checked in [the migration tracker](https://airbyte.metabaseapp.com/model#eyJuYW1lIjoiQ29ubmVjdG9yIEluc2lnaHRzIiwiZGVzY3JpcHRpb24iOm51bGwsImRhdGFzZXRfcXVlcnkiOnsiZGF0YWJhc2UiOjEwMCwidHlwZSI6InF1ZXJ5IiwicXVlcnkiOnsic291cmNlLXRhYmxlIjoiY2FyZF9fOTc5MCIsImZpbHRlciI6WyI9IixbImZpZWxkIiwiY29ubmVjdG9yX3RlY2huaWNhbF9uYW1lIix7ImJhc2UtdHlwZSI6InR5cGUvVGV4dCJ9XSwic291cmNlLWZyZXNoY2FsbGVyIl19fSwiZGlzcGxheSI6InRhYmxlIiwicGFyYW1ldGVycyI6W10sInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnsidGFibGUucGl2b3RfY29sdW1uIjoib3NzX3VzYWdlIiwidGFibGUuY2VsbF9jb2x1bW4iOiJjbG91ZF9zeW5jX3N1Y2Nlc3NfcmF0ZSJ9LCJvcmlnaW5hbF9jYXJkX2lkIjo5NzkwfQ==) filtered down to source-freshcaller, it's current CDK is 0.55.2.

So what could help here is to pin CDK to `0.55.2` in `pyproject.toml`, and re-lock. Let me try that.